error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -386,9 +386,13 @@ Optional<T> after(Optional<T> optional1, Optional<T> optional2) {
    */
   static final class OptionalIdentity<T> {
     @BeforeTemplate
+    @SuppressWarnings("NestedOptionals" /* Auto-fix for the `NestedOptionals` check. */)
```

Comment:
```suggestion
    @SuppressWarnings("NestedOptionals" /* Auto-fix for the `NestedOptionals` violation. */)
```
Not wrong, but this is an alternative.

Reply:
Hmm, I'm not feeling this suggestion, TBH. Elsewhere we use `/* This violation will be rewritten. */`, perhaps that would work? (If so, we should do the same for `OptionalOrOtherOptional`.)

Reply:
^ So far `/* This violation will be rewritten. */` is used only for SonarCloud violations. In e.g `ReactorRules` and `TimeRules` I see that `NestedPublishers` and `TimeZoneUsage` violations are suppressed with out a comment. Likely we should add some text in each case, but that's for another PR, then. Will drop it here.